### DISCUSSION TITLE
Pastas course at FH-DGGV conference in Aachen March 2024, Germany

### DISCUSSION THREAD

**Discussion by raoulcollenteur** [pastas/pastas#644] [Courses&Events]
Hi All,

On the 20th of March 2024 there will be another Pastas course at the "29. Tagung der Fachsektion Hydrogeologie e. V. in der DGGV" in Aachen, Germany. More info and registration here:

https://tagung2024.fh-dggv.de/fortbildungen.html

Looking forward to see you in Aachen!

Cheers,

Raoul

**Comment by AviadAvr** [comment]
Hi, is the workshop going to be in English?

If so, do you know if it's possible to sign up and paying for the workshop only, without paying for the entire conference?

Thanks!

**Reply by raoulcollenteur** [reply]
Hi @AviadAvr,

The workshop will probably be in German given the audience. It is possible to register without the conference, I think. 

Where are you based? We might organize an international workshop next year in Switzerland, The Netherlands, Belgium, or the US..

Cheers,
Raoul

**Reply by AviadAvr** [reply]
I'd be happy to join a workshop in NL / Belguim!
